core/modules/system/system.module

```diff
@@ -947,34 +921,90 @@ function system_menu() {
     'file' => 'system.admin.inc',
     'weight' => 20,
   );
+
+  // URL handling.
+  $items['admin/config/urls'] = array(
+    'title' => 'URL handling',
+    'description' => 'Settings related to URLs.',
```

Comment:
A more elaborate description might be better here rather than restating the title (though I can't think of a great description either).